sources/install.sh

```diff
@@ -1639,6 +1639,12 @@ function install_scout() {
   add-test-command "scout --help"
 }
 
+function install_azurecli(){
+  colorecho "Installing Azure cli"
+  fapt azure-cli
+  add-test-command "az --help"
```

Comment:
The best command would be : `az version` in order to be compliant with other tools.

Reply:
The test command has been updated to `az version`